web/src/components/card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Card() {
+    return (
+        <div className="absolute bg-white mx-4 md:ml-10 rounded-bl-lg w-11/12 md:w-5/12">
```

Comment:
I don't think you need `absolute` anymore. I'd also add `container` and drop `mx-4`.

Reply:
@asasmith adding the class container doesnt actually do anything to the card, mx-4 was the only way I was able to get align properly. Feel free to try yourself. this is what I get when I apply container and remove margin: 
<img width="805" alt="Screen Shot 2021-05-28 at 4 41 09 PM" src="https://user-images.githubusercontent.com/61708936/120039819-84af0780-bfd3-11eb-9d3b-7b66dfd450d7.png">

Reply:
Hmm, I thought  I had it working in the dev tools. Maybe I missed something.
If that's the case we should probably wrap this is in a div with the `container` class then. 

I don't think we want it to look like this vv
![Screen Shot 2021-05-28 at 4 52 55 PM](https://user-images.githubusercontent.com/11489171/120040789-297e1480-bfd5-11eb-8693-2b03279b5f77.png)

Reply:
I think this will look better (the nav needs some work as well).

![Screen Shot 2021-05-28 at 4 56 09 PM](https://user-images.githubusercontent.com/11489171/120041052-9db8b800-bfd5-11eb-9d08-7e3748aedb7a.png)

Reply:
Also, looking at these screenshots, you probably should add `padding-right` to match what's on the left side

Reply:
I'm merged an update to the tailwind config to center the `container` classes

Reply:
@asasmith That didnt work for me. Can you confirm what you have in your html? 
<img width="1170" alt="Screen Shot 2021-05-28 at 5 17 27 PM" src="https://user-images.githubusercontent.com/61708936/120043463-09048900-bfda-11eb-937d-46e9fa75140d.png">